ghost/core/core/server/data/schema/fixtures/fixtures.json

```diff
@@ -929,7 +929,7 @@
                     "recommendation": ["browse", "read"]
                 },
                 "Author": {
-                    "post": ["browse", "read", "add"],
```

Comment:
We should also update these in the test fixtures: https://github.com/TryGhost/Ghost/blob/cdf45172a7f63c0cf773abfbd31349393dda3782/ghost/core/test/utils/fixtures/fixtures.json#L1125

Reply:
yes, we should!!

Reply:
I've updated these, and I've updated them to be more explicit too - pulling out only the permissions we want, because there is an (unused) post:publish permission